commcid.go

```diff
@@ -93,6 +212,50 @@ func CIDToDataCommitmentV1(c cid.Cid) ([]byte, error) {
 	return commD, nil
 }
 
+// PieceMhCIDToDataCommitmentV1 extracts the raw data commitment and unpadded data size from the CID
+func PieceMhCIDToDataCommitmentV1(c cid.Cid) ([]byte, uint64, error) {
```

Comment:
@LexLuthr there's probably some harmonization of names and deciding how you want to look at this for you. e.g. this function might have any number of better names.

Feel free to choose whatever helps you (and the relevant folks in curio / lotus land) work with this in the easiest way.